shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -149,11 +149,31 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([method isEqualToString:@"Share.invoke"]) {
     [self showShareViewController:args];
     result(nil);
+  } else if ([method isEqualToString:@"ContextMenu.showSystemContextMenu"]) {
```

Comment:
@justinmc Wanna double check before landing it:

For non-text input, If developers setup `contextMenuBuilder` to use system menu, does it call this method? Does it crash or does it just NO-OP?

Reply:
Currently it calls it, even if there is no active text input connection. The result I see is that nothing happens. I wonder if there's anything we can do to prevent that unnecessary call, let me look on the framework side...

To be clear, at least the only way to do this now is to directly use SystemContextMenuController. It's not currently possible to use SystemContextMenu without an EditableText.

Reply:
I don't think there's a good way for the framework to know whether or not there is an active text input connection at that point (in SystemContextMenuController). Maybe should the ContextMenu.showSystemContextMenu call indicate an error in the case where it doesn't succeed in showing the menu?

Reply:
I'm adding some docs to the framework PR about this.

Reply:
> I don't think there's a good way for the framework to know whether or not there is an active text input connection at that point (in SystemContextMenuController). Maybe should the ContextMenu.showSystemContextMenu call indicate an error in the case where it doesn't succeed in showing the menu?

I think I can add a warning on the engine side.